src/ConfigCatClientOptions.ts

```diff
@@ -2,21 +2,31 @@ import { ConfigCatConsoleLogger } from "./ConfigCatLogger";
 import { IConfigCatLogger, IAutoPollOptions, ILazyLoadingOptions, IManualPollOptions, LogLevel } from "./index";
 import COMMON_VERSION from "./Version";
 
+
+/** Restrict the location of your feature flag and setting data within the ConfigCat CDN. */
```

Comment:
Control the location of the config.json files containing your feature flags and settings within the ConfigCat CDN.

Reply:
Fixed